app/test/frontend/golden/pkg_show_page.html

```diff
@@ -22,6 +22,13 @@
       ga('send', 'pageview');
     </script>
     <script async="" defer="" src="//www.google.com/insights/consumersurveys/async_survey?site=5wdvu4vc5lhop3xbatxof6lzcm"></script>
+    <meta name="twitter:card" content="summary" />
```

Comment:
If I ask you to fix the indent, will you be mad @isoos ?

Reply:
Either the template or the output sucks. If you are fine with that tradeoff, it can be indented.